### PR TITLE
Updates to Gitea Operator workload

### PR DESCRIPTION
**Pull request by wkulhanek**
##### SUMMARY

Updates for the Gitea Operator Workload to exploit the latest operator updates.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
ocp4_workload_gitea_operator

### CODE REVIEW

File: ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/defaults/main.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ ocp4_workload_gitea_operator_starting_csv: ""
 ocp4_workload_gitea_operator_catalog_image: quay.io/gpte-devops-automation/gitea-catalog
 
 # Catalog image tag
-ocp4_workload_gitea_operator_catalog_image_tag: v1.0.3
+ocp4_workload_gitea_operator_catalog_image_tag: latest
```
Comment (fridim) [inline ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/defaults/main.yml:29]: I think we always want to pin-point version, it's part of our contribution guidlines?

File: ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/tasks/workload.yml [hunk 2]
```diff
@@ -45,122 +46,84 @@
...
+  - name: Set up users in Gitea
+    when: ocp4_workload_gitea_operator_create_users | bool
+    block:
+    - name: Create the users in Gitea
+      include_tasks: create_user.yml
+      loop: "{{ range(1, ocp4_workload_gitea_operator_user_number | int + 1, 1) | list }}"
+
+    - name: Migrate the repositories for the Gitea users
+      include_tasks: migrate_repos.yml
+      loop: "{{ (range(1, ocp4_workload_gitea_operator_user_number | int + 1, 1) | list) | product(ocp4_workload_gitea_repositories_list) | list }}"
```
Comment (fridim) [inline ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/tasks/workload.yml:98]: Aeration :)

```yaml
      loop: >-
        {{ 
        ( range(1, ocp4_workload_gitea_operator_user_number | int + 1, 1) | list )
        | product(ocp4_workload_gitea_repositories_list)
        | list
        }}
```

File: ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/defaults/main.yml (hunk 1 above)
Comment (wkulhanek) [reply ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/defaults/main.yml:29]: Good point.

**Review by fridim — APPROVED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/tasks/workload.yml (hunk 2 above)
Comment (wkulhanek) [reply ansible/roles_ocp_workloads/ocp4_workload_gitea_operator/tasks/workload.yml]: Done.

**Pull request merged**